File: DAL/Entities/ShopCartItem.fs
namespace DAL_Models

open System
open System.ComponentModel.DataAnnotations

[<CLIMutable>]
type ShopCartItem = {
    
    [<Key>]
    Id:Guid
    
    Entity: Catalog_Entity
    Quanity: uint
    UserName: string
    
    OrderId: int
    Order: Catalog_Order
    
}

File: DAL/Models/ShopCartItem.cs
using System;

namespace Models
{
    public class ShopCartItem
    {
        public Guid Id { get; set; }
        public Catalog_Entity Entity { get; set; }
        public uint Quanity { get; set; }
        public string UserName { get; set; }

        public Catalog_Order Order { get; set; }
        public int? OrderId { get; set; }
    }
}
